pkg/resource/db_instance/sdk.go

```diff
@@ -1775,7 +1775,6 @@ func (rm *resourceManager) sdkUpdate(
 	// For dbInstance inside dbCluster, it's either aurora or
 	// multi-az cluster case, in either case, the below params
 	// are not controlled in instance level.
-	// multi-az cluster control it on cluster leve, aurora doesn't need it
```

Comment:
Any idea what this is?

Reply:
looks like it's a build/merge issue, this comment is removed in template  https://github.com/aws-controllers-k8s/rds-controller/blob/main/templates/hooks/db_instance/sdk_update_post_build_request.go.tpl#L30 and should be removed here as well